.github/workflows/update-hash.yml

```
@@ -2,6 +2,10 @@ name: Update nix hashes
 
 on:
   push:
+    branches-ignore:
+      - master
+    tags-ignore:
+      - '*'
```

```suggestion
      - '**'
```
Matches all characters, also `/`. @nomeata Probably we'll never have slashes in tag names, but this may be more correct.